product_docs/docs/biganimal/release/using_cluster/05_monitoring_and_logging/monitoring_from_aws/index.mdx

```diff
@@ -0,0 +1,128 @@
+---
+title: "Viewing metrics and logs from AWS"
+---
+
+BigAnimal sends all metrics and logs from PostgreSQL clusters to AWS. You can view metrics and logs directly from AWS using AWS CloudWatch. The following describes how these logs and metrics can be accessed.
+
+## Logs
+
+### AWS CloudWatch
+
+When BigAnimal deploys workloads on AWS, the logs from the PostgreSQL clusters are forwarded to AWS CloudWatch [Log groups as Log streams](https://docs.aws.amazon.com/AmazonCloudWatch/latest/logs/Working-with-log-groups-and-streams.html#ViewingLogData).
```

Comment:
Wondering if we should use Postgres here instead of PostgreSQL. That is our convention when we are referring generically to either community PostgreSQL or EPAS. If so, there are other places in these topics where we should consider using the same convention.